chapters/VK_EXT_acquire_wl_display/acquire_wl_display.txt

```diff
@@ -0,0 +1,66 @@
+[open,refpage='vkAcquireWaylandDisplayEXT',desc='Acquire access to a VkDisplayKHR using Wayland',type='protos']
+--
+
+To acquire permission to directly access a display in Vulkan from a Wayland
+compositor, call:
+
+include::{generated}/api/protos/vkAcquireWaylandDisplayEXT.txt[]
+
+  * pname:physicalDevice The physical device the display is on.
+  * pname:display A wl_display connected to the Wayland compositor that
+	currently owns pname:display.
+  * pname:manager A zwp_drm_lease_manager_v1 connected to the Wayland compositor
+	that owns the connectors requested for lease.
+  * pname:nConnectors The number of valid pointers in the pname:pConnectors
+	array.
+  * pname:pConnectors Array of slink:VkWaylandLeaseConnectorEXT which are being
+	leased.
+
+All permissions necessary to control the display are granted to the Vulkan
+instance associated with pname:physicalDevice until the display is released, the
+Wayland connection specified by pname:display is terminated, or the compositor
+revokes the lease.
+Permission to access the display may: be temporarily revoked during periods
+when the Wayland compositor from which control was acquired itself looses access
+to pname:display.
+During such periods, operations which require access to the display must:
+fail with an approriate error code.
+If the Wayland server declines to grant the lease or the granted lease does not
+include the requested connectors, the call must: return the error code
+ename:VK_ERROR_INITIALIZATION_FAILED.
+
+[NOTE]
+.Note
+====
+One example of when a Wayland compositor loses access to a display is when it
+loses ownership of its virtual terminal.
+====
+
+include::{generated}/validity/protos/vkAcquireWaylandDisplayEXT.txt[]
+--
+
+[open,refpage='VkWaylandLeaseConnectorEXT',desc='Resource for relating Wayland connectors to VkDisplayKHRs',type='structs']
+--
+
+The sname:VkWaylandLeaseConnectorEXT structure is defined as:
+
+include::{generated}/api/structs/VkWaylandLeaseConnectorEXT.txt[]
```

Comment:
You have an opening of a refpage for VkWaylandLeaseConnectorEXT, and the generated API include, but no generated validity include. `script/check_spec_links.py  chapters/VK_EXT_acquire_wl_display/acquire_wl_display.txt appendices/VK_EXT_acquire_wl_display.txt ` (which I recommend running) reports:

```
chapters/VK_EXT_acquire_wl_display/acquire_wl_display.txt:47:0: warning: Saw /api/ include for VkWaylandLeaseConnectorEXT, but no matching /validity/ include (-Wmissing_validity_include)
                                                                          Expected a line with include::{generated}/validity/structs/VkWaylandLeaseConnectorEXT.txt[]

include::{generated}/api/structs/VkWaylandLeaseConnectorEXT.txt[]
^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
Nevermind, figured it out.